package.json

```diff
@@ -1,71 +1,31 @@
 {
   "name": "ember-test-selectors",
-  "version": "7.0.0",
+  "private": true,
   "description": "Enabling better Test selectors in Ember.js applications.",
-  "keywords": [
-    "ember-addon"
-  ],
   "license": "MIT",
   "author": "Mainmatter GmbH",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
   "repository": "https://github.com/mainmatter/ember-test-selectors",
+  "workspaces": [
+    "ember-test-selectors",
+    "tests/*"
+  ],
```

Comment:
I don't think you need this when you have `pnpm-workspace.yaml` to define your workspaces 🤔